RecoTracker/SpecialSeedGenerators/python/CombinatorialSeedGeneratorForCosmicsP5_cff.py

```diff
@@ -82,18 +81,18 @@
                                                   combinatorialcosmicseedingpairsTECnegP5)
 combinatorialcosmicseedinglayersP5 = cms.Sequence(combinatorialcosmicseedinglayersP5Task)
 #recHitMatcher
```

Comment:
Since you removed the corresponding (commented out) include, you can also remove this comment line that referred to it